narwhals/_sql/expr.py

```diff
@@ -181,14 +181,18 @@ def _function(self, name: str, *args: NativeExprT | PythonLiteral) -> NativeExpr
     def _lit(self, value: Any) -> NativeExprT:
         return self.__narwhals_namespace__()._lit(value)
 
-    def _when(self, condition: NativeExprT, value: NativeExprT) -> NativeExprT:
-        return self.__narwhals_namespace__()._when(condition, value)
-
     def _coalesce(self, *expr: NativeExprT) -> NativeExprT:
         return self.__narwhals_namespace__()._coalesce(*expr)
 
     def _count_star(self) -> NativeExprT: ...
 
+    def _when(
+        self,
+        condition: NativeExprT,
+        value: NativeExprT,
+        otherwise: NativeExprT | None = None,
+    ) -> NativeExprT: ...
```

Comment:
instead of `: ...`, can we make this one return
```
return self.__narwhals_namespace__()._when(condition, value, otherwise)
```

and then add update the definition in `_sql/namespace.py` to have `otherwise`?